lib/src/linter.dart

```diff
@@ -325,9 +331,11 @@ class SourceLinter implements DartLinter, AnalysisErrorListener {
 
   @override
   Iterable<AnalysisErrorInfo> lintPubspecSource(
-      {String contents, String sourceUrl}) {
+      {String contents, String sourcePath}) {
     var results = <AnalysisErrorInfo>[];
 
+    Uri sourceUrl = sourcePath == null ? null : p.toUri(sourcePath);
```

Comment:
`url` is consistent with the `yaml` API I eventually call and that was my thinking but I tend to agree!